app/script/event/Client.js

```diff
@@ -36,6 +36,7 @@ z.event.Client = {
     LOCATION: 'conversation.location',
     MESSAGE_DELETE: 'conversation.message-delete',
     MESSAGE_HIDDEN: 'conversation.message-hidden',
+    MESSAGE_TOO_BIG: 'conversation.message-too-big',
```

Comment:
We have different limits for incoming & outgoing messages so maybe better to call it `INCOMING_MESSAGE_TOO_BIG`.